src/environment.js

```diff
@@ -0,0 +1,40 @@
+export class Environment {
+  /**
+   * @param {String} environment overwritten environment
+   * @param {Object} base overwrite window to get location
+   */
+  constructor(environment = '', base = window.location.href) {
+    let selectedEnvironment = 'production';
+    if (!environment) {
+      if (base.includes('localhost')) {
+        selectedEnvironment = 'development';
+      }
+      if (base.includes('dev')) {
+        selectedEnvironment = 'development';
+      }
+      if (base.includes('uat')) {
+        selectedEnvironment = 'uat';
+      }
+    } else {
+      selectedEnvironment = environment;
+    }
+
+    switch (selectedEnvironment) {
+      case 'development': {
+        this.url = 'https://api.forms.platforms.qld.gov.au/fesrqwsyzlbtegd/';
```

Comment:
can these be brought to a config file at the root and dropped in on build?

Reply:
It could be, but I don't know what that gives?